packages/maskbook/src/plugins/Airdrop/base.ts

```diff
@@ -0,0 +1,15 @@
+import type { Plugin } from '@dimensiondev/mask-plugin-infra'
+import { PLUGIN_ID, PLUGIN_ICON, PLUGIN_NAME, PLUGIN_DESCRIPTION } from './constants'
+
+export const base: Plugin.Shared.Definition = {
+    ID: PLUGIN_ID,
+    icon: PLUGIN_ICON,
+    name: { fallback: PLUGIN_NAME },
+    description: { fallback: PLUGIN_DESCRIPTION },
+    publisher: { name: { fallback: 'Mask Network' }, link: 'https://mask.io/' },
+    enableRequirement: {
+        architecture: { app: false, web: true },
+        networks: { type: 'opt-out', networks: {} },
+        target: 'insider',
```

Comment:
You changed its visibility from `public, web-only` (before) to `insider, web-only`. Is that intended?

Reply:
To change it to `stable` is OK.